tests/baselines/reference/contextuallyTypedParametersWithQuestionToken.types

```diff
@@ -0,0 +1,55 @@
+//// [tests/cases/compiler/contextuallyTypedParametersWithQuestionToken.ts] ////
+
+=== contextuallyTypedParametersWithQuestionToken.ts ===
+// https://github.com/microsoft/TypeScript/issues/54948
+
+const f1: (a: string, b: number) => void = function self(a, b?) {
+>f1 : (a: string, b: number) => void
+>a : string
+>b : number
+>function self(a, b?) {  b;  self("");  self("", undefined);} : (a: string, b?: number) => void
+>self : (a: string, b?: number) => void
+>a : string
+>b : number
+
+  b;
+>b : number

```

Comment:
Maybe I misunderstood the bug, but shouldn't that be `>b : number | undefined`?

Reply:
You got it right but it turned out that I only fixed it in one spot - for the parameter type in the signature but not for the parameter type within the function's body. Fixed it and thanks for noticing this!